src/util/articleUtil.js

```diff
@@ -23,13 +23,13 @@ export const transformArticleToApiVersion = article => ({
   },
 });
 
-export const transformArticle = (article, locale) => {
+export const transformArticle = article => {
   if (!article) return undefined;
   const footNotes = defined(article.metaData.footnotes, []);
   return {
     ...article,
-    created: formatDate(article.created, locale),
-    updated: formatDate(article.updated, locale),
+    created: formatDate(article.created, 'nb'),
+    updated: formatDate(article.updated, 'nb'),
```

Comment:
Viss man alltid skal vise på norsk format, hvorfor trenger vi å støtte andre localer i formatDate?

Reply:
Kanskje defaulte til 'nb'? Eller skal jeg fjerne muligheten for å sende inn en annen locale?